lib/ardb.rb

```diff
@@ -43,6 +46,10 @@ def self.require_db_file
     rescue LoadError
       require File.expand_path(ENV['ARDB_DB_FILE'], ENV['PWD'])
     end
+  rescue LoadError
+    raise InvalidDBFileError, "can't require `#{ENV['ARDB_DB_FILE']}`, " \
+                              "try setting the ARDB_DB_FILE env var with the " \
+                              "file path to your db file"
```

Comment:
@jcredding this is a good start to this message.  I like the "can't require..." part a lot.  Maybe say ", check that the ARDB_DB_FILE env var is set to file path of your db file" for the second part?